cli/internal/cmd/init.go

```diff
@@ -259,7 +265,47 @@ func (d *initDoer) Do(ctx context.Context) error {
 	if err != nil {
 		return &nonRetriableError{fmt.Errorf("init call: %w", err)}
 	}
-	d.resp = resp
+
+	res, err := resp.Recv() // get first response, either success or failure
```

Comment:
Just as a heads up: I think it is cleaner to first check the error and then switch over the message type of `res`. 
For an example you can see the last code snippet here: https://protobuf.dev/reference/go/go-generated/#oneof